modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -192,6 +192,15 @@ def edipi_check
             'Please submit an issue at ask.va.gov or call 1-800-MyVA411 (800-698-2411) for assistance.')
         end
       end
+
+      def claims_v1_logging(icn, request, poa = nil)
```

Comment:
I haven't confirmed, but do we need to pass `request` in? Shouldn't the `application_controller` have access to that? Also, datadog should add that for us, so we shouldn't need it anywho? Also should we log the cid and/or label or just the cid?

Reply:
What do you mean by "should we log the cid and/or label..."? the cid is logged below on line 200